test/functional/tensor/vector/tests/test-tensor-vector-ForallVectorRef1d.hpp

```diff
@@ -54,14 +71,39 @@ void ForallVectorRef1dImpl()
   }
 
 
+  // evaluate complex left side division on all() range
   for(size_t i = 0;i < N; ++ i){
     C[i] = 0.0;
   }
 
+  tensor_copy_to_device<policy_t>(C_ptr, C);
+
+  tensor_do<policy_t>([=] RAJA_HOST_DEVICE (){
+    Z_d[all] = 3 + ((X_d[all]*Y_d[all])/Y_d[all]) + 9;
+  });
+
+  tensor_copy_to_host<policy_t>(C, C_ptr);
+
+  for(size_t i = 0;i < N;i ++){
+    ASSERT_SCALAR_EQ(element_t(3+((A[i]*B[i])/B[i]))+9, C[i]);
```

Comment:
Should these expressions look more like this `3 + (X[i]*(5/Y[i])) + 9`?

Reply:
It should look more like the problems encountered here https://github.com/LLNL/RAJA/issues/1303. I'll add an extra vector in there so that it can look more like `(3+(X[i]*W[i])/Y[i]) + 9`.